src/Console/ConsoleOutput.php

```diff
@@ -77,40 +76,40 @@ public function logBadMsiErrorMessage(MetricsCalculator $metricsCalculator, floa
         $this->io->error(
             sprintf(
                 self::CI_FLAG_ERROR,
-                ($type === TestRunConstraintChecker::MSI_FAILURE ? 'MSI' : 'Covered Code MSI'),
+                ($type === MinMsiChecker::MSI_FAILURE ? 'MSI' : 'Covered Code MSI'),
                 $minMsi,
-                ($type === TestRunConstraintChecker::MSI_FAILURE ?
+                ($type === MinMsiChecker::MSI_FAILURE ?
                     $metricsCalculator->getMutationScoreIndicator() :
                     $metricsCalculator->getCoveredCodeMutationScoreIndicator()
                 )
             )
         );
     }
 
-    /**
-     * @throws InvalidTypeException
-     */
-    public function logMinMsiCanGetIncreasedNotice(MetricsCalculator $metricsCalculator, float $minMsi, string $type): void
+    public function logMinMsiCanGetIncreasedNotice(float $minMsi, float $msi): void
     {
-        if ($type !== TestRunConstraintChecker::MSI_OVER_MIN_MSI && $type !== TestRunConstraintChecker::COVERED_MSI_OVER_MIN_MSI) {
-            throw InvalidTypeException::create($type);
-        }
+        $typeString = 'MSI';
 
-        if ($type === TestRunConstraintChecker::MSI_OVER_MIN_MSI) {
-            $typeString = 'MSI';
-            $msi = $metricsCalculator->getMutationScoreIndicator();
-        } else {
-            $typeString = 'Covered Code MSI';
-            $msi = $metricsCalculator->getCoveredCodeMutationScoreIndicator();
-        }
+        $this->io->note(
+            sprintf(
+                self::MIN_MSI_CAN_GET_INCREASED_NOTICE,
+                $typeString,
+                $msi - $minMsi,
+                $typeString,
+                $typeString
+            )
+        );
+    }
 
-        $msiDifference = $msi - $minMsi;
+    public function logMinCoveredCodeMsiCanGetIncreasedNotice(float $minMsi, float $coveredCodeMsi): void
+    {
+        $typeString = 'Covered Code MSI';
 
         $this->io->note(
             sprintf(
                 self::MIN_MSI_CAN_GET_INCREASED_NOTICE,
                 $typeString,
-                $msiDifference,
```

Comment:
IMO, removing this variable makes the code less understandable

Reply:
Fair, i'll introduce it back, but after since I'm going to get rid of most of those in the coming PR after rebase :)